src/main/java/frc/robot/maps/ColdStart.java

```diff
@@ -123,32 +132,54 @@ public SwerveDriveMap getDriveMap() {
     public ElevatorMap getElevatorMap() {
         CSSparkFlex leftMotor = new CSSparkFlex(11);
         CSSparkFlex rightMotor = new CSSparkFlex(12);
-        SparkFlexConfig config = new SparkFlexConfig();
-        config.follow(leftMotor.getMotorController());
-        rightMotor.getMotorController().configure(config, ResetMode.kNoResetSafeParameters,
+        SparkFlexConfig configRight = new SparkFlexConfig();
+        SparkFlexConfig configLeft = new SparkFlexConfig();
+
+        configRight.follow(leftMotor.getMotorController());
+        configRight.voltageCompensation(11.5);
+        configRight.smartCurrentLimit(30);
+        rightMotor.getMotorController().configure(configRight, ResetMode.kResetSafeParameters,
                 PersistMode.kPersistParameters);
 
-        CSEncoder encoder = new CSEncoder(2, 3);
+        configLeft.voltageCompensation(11.5);
+        configLeft.smartCurrentLimit(30);
+        // Gear reduction is 22.2 sprocket diameter is 1.75 inches
+        configLeft.encoder.positionConversionFactor((1 / 22.2) * Math.PI * 1.75);
```

Comment:
Probably worth setting the `velocityConversionFactor` as well. (Same value divided by 60 I think?)